crates/fuel-core/src/executor.rs

```diff
@@ -2626,7 +2625,7 @@ mod tests {
     fn contracts_balance_and_state_roots_no_modifications_updated() {
         // Values in inputs and outputs are random. If the execution of the transaction successful,
         // it should actualize them to use a valid the balance and state roots. Because it is not
-        // changes, the balance the root should be default - `fuel_merkle::common::empty_sum_sha256()`.
```

Comment:
This was actually incorrect: the spec states that this should be a 32 byte 0 for any tree with height greater than 0. See https://github.com/celestiaorg/celestia-specs/blob/master/src/specs/data_structures.md#sparse-merkle-tree.